talk/setup.tex

```diff
@@ -80,7 +80,7 @@
 \usepackage{tikz-uml}
 
 \usepackage{booktabs}
-\usepackage{ifthen}
+\usepackage{comment}
```

Comment:
I think this will break the PR about the index of good practices and index of exercises etc. That's not a showstopper, though. I can re-introduce `ifthen` if necessary.

Reply:
I had to switch over to the commend package for including/excluding content, because it can handle verbatim environments inside the commented section.

Reply:
You can still use the `ifthen` package if you want! Just readd it in your PR!